src/bundle/Controller/Permission/LanguageLimitationController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\Bundle\AdminUi\Controller\Permission;
+
+use Ibexa\AdminUi\Permission\LimitationResolverInterface;
+use Ibexa\Contracts\AdminUi\Controller\Controller;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\Content\Location;
+use Ibexa\Contracts\Core\Repository\Values\Content\VersionInfo;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Response;
+
+final class LanguageLimitationController extends Controller
+{
+    private LimitationResolverInterface $limitationResolver;
+
+    public function __construct(LimitationResolverInterface $limitationResolver)
+    {
+        $this->limitationResolver = $limitationResolver;
+    }
+
+    public function loadLanguageLimitationsForContentAction(
+        Content $content,
```

Comment:
In my opinion it would be better to load `ContentInfo` instead. `Content` objects are heavy and as long as you don't need to specifically use fields, `ContentInfo` should be enough.

Reply:
Fixed: https://github.com/ibexa/admin-ui/pull/887/files#diff-a20ff425f5a6cb0616053be3fba6bc54c06e5a0259e7d4b03cc045818583f66bR63